App/Controllers/IndexController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controllers;
+
+use Core\View;
+use App\Models\User;
+use App\Models\Task;
+
+class IndexController {
+
+    public function index()
+    {
+        if(session_status() !== PHP_SESSION_ACTIVE)
```

Comment:
Might be a good idea here to create a SessionHandler class that wil do this kind of checks and retrievals of some data from $_SESSION.